tests/wpt/meta/streams/transform-streams/errors.any.js.ini

```diff
@@ -0,0 +1,34 @@
+[errors.any.sharedworker.html]
+  expected: ERROR
+
+[errors.https.any.shadowrealm-in-serviceworker.html]
+  expected: ERROR
+
+[errors.any.shadowrealm-in-sharedworker.html]
+  expected: ERROR
+
+[errors.any.shadowrealm-in-shadowrealm.html]
+  expected: ERROR
+
+[errors.any.serviceworker.html]
+  expected: ERROR
+
+[errors.https.any.shadowrealm-in-audioworklet.html]
+  expected: ERROR
+
+[errors.any.shadowrealm-in-dedicatedworker.html]
+  expected: ERROR
+
+[errors.any.shadowrealm-in-window.html]
+  expected: ERROR
+
+[errors.any.worker.html]
+  expected: ERROR
+  [abort should set the close reason for the writable when it happens before cancel during start, and cancel should reject]
+    expected: FAIL
+
+
+[errors.any.html]
+  expected: ERROR
```

Comment:
Note that the suite is still erroring, due to an unhandled promise rejection... We should fix that or note it in an issue.